feed/index.html

```diff
@@ -3,15 +3,21 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Description" content="This is a feed page where all the user's posts are displayed.">
     <link rel="stylesheet" href="../dist/css/index.css">
-        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
-    <title>Profile</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/dompurify/3.0.6/purify.min.js" integrity="sha512-H+rglffZ6f5gF7UJgvH4Naa+fGCgjrHKMgoFOGmcPTRwR6oILo5R+gtzNrpDp7iMV3udbymBVjkeZGNz1Em4rQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/dompurify/3.0.6/purify.min.js.map"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <script src="../src/scripts/authenticatedGuard.js"></script>
+    <script src="./../src/scripts/feed.js" type="module"></script>
+    <script src="./../src/scripts/logout.js" type="module"></script>
+    <title>Feeds/All Posts</title>
 </head>
 <body>
     
     <nav class="navbar navbar-expand-lg bg-dark">
         <div class="container">
-            <a class="navbar-brand" href="#">Dear Diary</a>
+            <a class="navbar-brand" href="/profile/index.html">DearDiary</a>
```

Comment:
```suggestion
            <a class="navbar-brand" href="/feed">DearDiary</a>
```
I think it makes more sense to have the navbar-brand anchor go to the home page. In this case that's the /feed page. Make sure to set or change this on all pages where the navbar is shown.

Reply:
Thank you for your suggestion, I will fix it